benchmarks/transactions-generator/src/lib.rs

```diff
@@ -106,18 +108,18 @@ impl TxGenerator {
             })),
         };
 
-        let transactions = Self::start_transactions_loop(
-            &self.params,
-            client_sender,
-            view_client_sender.clone(),
-            runner_state.clone(),
-        )?;
-
-        let block_updates = Self::start_block_updates(view_client_sender, runner_state.clone());
+        for _ in 0..self.params.thread_count {
+            self.runner.push(Self::start_transactions_loop(
+                &self.params,
+                client_sender.clone(),
+                view_client_sender.clone(),
+                runner_state.clone(),
+            )?);
+        }
 
-        let report_updates = Self::start_report_updates(runner_state);
+        self.runner.push(Self::start_block_updates(view_client_sender, runner_state.clone()));
+        self.runner.push(Self::start_report_updates(runner_state));
```

Comment:
should we store these in a different member than `runners`?  Not sure if that will help improve readability.